src/cljdoc/util/fixref.clj

```diff
@@ -76,6 +84,12 @@
   (let [doc     (Jsoup/parse html-str)
         scm-rev (or (:name (:tag scm))
                     (:commit scm))]
+    (doseq [ext-link (->> (.select doc "a")
+                          (map #(.attributes %))
+                          (remove #(own-uri? (.get % "href")))
```

Comment:
I would reorder the `filter` to come before the `remove`. That way you're saying: "filter out everything that isn't an absolute URI, then additionally remove anything pointing to cljdoc. Generally, I will try and put the broader filter before a narrower one.

Reply:
Oh, good point, thanks! I'm gonna push an update now.